package.json

```diff
@@ -1,19 +1,15 @@
 {
   "name": "federalist-uswds-jekyll",
-  "version": "1.0.0",
+  "version": "1.3.0",
   "description": "Federalist using USWDS+Jekyll",
-  "main": "src/js/app.js",
   "scripts": {
-    "build": "npm run build:uswds; npm run build:js; npm run build:jekyll",
+    "build": "npm run build:uswds; npm run build:jekyll",
     "build:jekyll": "bundle exec jekyll build",
-    "build:js": "mkdir -p assets/js; cp -R src/js/* assets/js/",
-    "build:uswds": "npm run build:uswds:assets; npm run build:uswds:sass",
-    "build:uswds:assets": "mkdir -p assets/uswds; cp -R node_modules/uswds/dist/css node_modules/uswds/dist/img node_modules/uswds/dist/fonts node_modules/uswds/dist/js assets/uswds/",
-    "build:uswds:sass": "mkdir -p _sass/uswds; cp -R node_modules/uswds/dist/scss/* _sass/uswds/",
-    "clean": "rm -rf assets/uswds; rm -rf _sass/uswds; rm -rf _site",
-    "federalist": "npm run build:uswds; npm run build:js",
+    "build:uswds": "ruby build.rb uswds;",
+    "clean": "ruby build.rb clean",
+    "federalist": "npm run build:uswds;",
     "postinstall": "bundle install",
-    "start": "bundle exec jekyll serve",
+    "start": "npm run build; bundle exec jekyll serve;",
     "test": "bundle exec htmlproofer _site; npx a11y '_site/**/*.html'"
   },
   "author": "",
```

Comment:
I'm a broken record as always, but at this point we're only using npm to download USWDS and as a task runner which doesn't seem necessary to me. I understand what you're saying about the template being ready for expansion, but adding jekyll-assets confuses this since it handles stuff one may use node for. I think we should either actually use node or don't use it. Anyone comfortable enough to use it at all, can probably handle creating their own package.json...

Reply:
I hear ya. But, as long as we do have a node dependency, then having a combined call makes sense. I default to using "npm run start" more than "npm install; npm run build:uswds; bundle exec jekyll serve"